examples/with-cookie-auth-fauna/utils/auth.js

```diff
@@ -1,46 +1,21 @@
 import { useEffect } from 'react'
 import Router from 'next/router'
-import cookie from 'cookie'
-import { FAUNA_SECRET_COOKIE } from './fauna-auth'
 
 export const login = ({ email }) => {
   window.localStorage.setItem('loggedInUser', email)
```

Comment:
With the change to remove auth() below, I believe nowhere else is using localstorage's 'loggedInUser'. It may still be useful to have for the frontend to be able to refer check if user is logged in or not (for UI elements for example) or can be removed to simplify.

Reply:
Removed. The less user info available in the client, the better. Nice catch.